handshake_client.go

```diff
@@ -891,10 +901,15 @@ func (c *Conn) verifyServerCertificate(certificates [][]byte) error {
 		// [UTLS SECTION START]
 		opts := x509.VerifyOptions{
 			Roots:         c.config.RootCAs,
-			CurrentTime:   c.config.time(),
 			Intermediates: x509.NewCertPool(),
 		}
 
+		if c.config.InsecureSkipTimeVerify {
+			opts.CurrentTime = certs[0].NotAfter
```

Comment:
This line looks a bit hacky but I have no better idea on it either.

Reply:
I agree. 
This is just a reference to `if c.config.time().After(serverCert.NotAfter) {` in the `loadSession` method